FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Aleksandr Honcharov on 01.06.2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+enum FeedImagesMapper {
+	private struct Root: Decodable {
+		let items: [Image]
```

Comment:
The `items` property is not used outside of `Root`, so it can be private.
```suggestion
		private let items: [Image]
```